calico_kubernetes/tests/kube_plugin_test.py

```diff
@@ -108,11 +116,104 @@ def test_delete(self):
 
             # Assert
             m_container_remove.assert_called_once_with()
-            self.assertEqual(namespace, self.plugin.namespace)
-            self.assertEqual(pod_name, self.plugin.pod_name)
-            self.assertEqual(docker_id, self.plugin.docker_id)
-            self.assertEqual(profile_name, self.plugin.profile_name)
-            m_datastore_client.remove_profile(profile_name)
+            assert_equal(namespace, self.plugin.namespace)
+            assert_equal(pod_name, self.plugin.pod_name)
+            assert_equal(docker_id, self.plugin.docker_id)
+            assert_equal(profile_name, self.plugin.profile_name)
+            self.m_datastore_client.remove_profile(profile_name)
+
+    def test_delete_error(self):
+        """Test Pod Deletion Hook Failure"""
+        """
+        If the datastore remove_profile function returns KeyError, the profile is not in the datastore.
+        Issue warning log, but do not fail.
```

Comment:
Can you rework this comment please? Characters are cheap, so let's have some full sentences. Something like "For this failure case we expect a warning log, but delete() should not raise an exception". (I found this sentence a bit hard to follow with my "knows nothing about the code" hat on).